ci-operator/jobs/redhat-performance/benchmark-runner/redhat-performance-benchmark-runner-main-postsubmits.yaml

```diff
@@ -1,7 +1,7 @@
 postsubmits:
   redhat-performance/benchmark-runner:
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
It's suggest to make the change in config (ci-operator/config/org_name/repo_name), rather than in jobs.